d2l-dom-focus.html

```diff
@@ -195,19 +195,49 @@
 
 		},
 
-		isFocusable: function(node, includeHidden) {
+		getPreviousFocusableAncestor: function(node, includeHidden, includeTabbablesOnly) {
+
+			if (!node) {
+				return null;
+			}
+
+			if (includeHidden === undefined) {
```

Comment:
To explicitly initialize it to `true` in the caller rather than depending on `isFocusable` to? Or to use a different value than what `isFocusable` will currently set? I'm fine with the former; didn't do the latter to keep consistency across the API.